docs/apis-clients/web-modeler-api/index.md

```diff
@@ -0,0 +1,78 @@
+---
+id: index
+title: Web Modeler API (beta) (REST)
+description: "Web Modeler Public API (beta) is a REST API and provides access to Web Modeler Data.
+Requests and responses are in JSON notation."
+---
+
+:::caution Beta Offering
+Web Modeler API is currently offered as a [beta release](../../reference/early-access#beta). It is not recommended for production use and there is no maintenance service guaranteed.
+
+Special [terms & conditions](https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/) apply.
+
+While in beta the API may introduce breaking changes without prior notice.
+
+However, we encourage you to provide feedback via your designated support channel or the [Camunda Forum](https://forum.camunda.io/).
+:::
+
+## OpenAPI documentation
+
+A detailed API description is also available as [OpenAPI](https://www.openapis.org/) specification at [https://modeler.cloud.camunda.io/swagger-ui/index.html](https://modeler.cloud.camunda.io/swagger-ui/index.html)
+
+## Authentication
+
+To authenticate for the API you need to generate a JWT token and pass it in each request.
+
+### Obtain JWT token
+
+Web Modeler provides a REST API under the endpoint `/api`. Clients can access this API using a JWT access token in an authorization header `Authorization: Bearer <JWT>`.
+
+**Example:**
+
+1. [Create an API Client](../../../components/console/manage-clusters/manage-api-clients/#create-a-client).
+2. Add permissions to this client for **Web Modeler Public API (beta)**.
+3. After creating the client, you can download a shell script to obtain a token
+4. When you run it, you will get something like the following:
+   ```json
+   {
+     "access_token": "eyJhbG...",
+     "expires_in": 300,
```

Comment:
No. I am not sure refresh tokens are a thing for public APIs. In UIs it's different, because you have an open website or a session. So you have something where you can register a timer and refresh the token.

With M2M clients this is less common. Usually, you assume stateless and keep the token for as long as it is valid. Getting a new token vs. refreshing a token also makes little difference, technically. So I would not describe this, unless there is a need for it.